### ISSUE TITLE
Error?

### ISSUE THREAD

**Issue by ItsaGamer05** [LegitCP/CPI-Bot#1]
Whenever I try to type "java -jar CPI-Bot-v1.0-SNAPSHOT.jar" into the command prompt, an error shows up: "Error: Unable to access jarfile CPI-Bot-v1.0-SNAPSHOT.jar" Please help!

**Comment by ItsGravix** [comment]
@ItsaGamer05 Try making a .bat file, put ```java -jar CPI-Bot-v1.0-SNAPSHOT.jar``` on it, and run the .bat file on the same folder where the ```CPI-Bot-v1.0-SNAPSHOT.jar``` file is.

**Comment by ItsaGamer05** [comment]
Yes, i tried doing that in the CPI-Bot-master.zip folder but it still wont work!
